frontend/src/components/3.Preferences/Preferences.js

```diff
@@ -53,7 +53,7 @@ function Preferences() {
   async function handleSearch() {
     // console.log(Coordinates);
     const data = await getNearbyRestaurants(
-        Coordinates, Distance, 'european');
+        Coordinates, Distance, 'european', Price);
```

Comment:
> I noticed that the <option> tags in this file on lines 162-165 set string values for the prices. Does the API call set the price correctly if the maxPriceLevel is set to a string value?
> 
> Maybe it would be better to change it? So the option values would be numbers, e.g. `<option value={1}>` instead of `<option value='1'>`

Just copying my comment from the previous PR

Reply:
![image](https://user-images.githubusercontent.com/49629647/114259088-8ae72580-9a1f-11eb-850a-a39b5fcc6987.png)
Good point. I think this is fine since before the request, the price is converted from string to number.